packages/meteor/helpers.js

```diff
@@ -144,8 +141,7 @@ Meteor.wrapAsync = function (fn, context) {
       if (Meteor.isClient) {
         callback = logErr;
       } else {
-        var fut = new Future();
-        callback = fut.resolver();
+        callback = () => console.error("Tried to use wrapAsync.");
```

Comment:
If not passing a callback on the server is no longer supported, it should probably throw an error here, or do the same thing as on the client and use logErr.